src/main/java/algorithms/minimumSpanningTree/kruskal/Edge.java

```diff
@@ -0,0 +1,36 @@
+package algorithms.minimumSpanningTree.kruskal;
```

Comment:
our other files typically place helper class in the same file. For consistency sake, we could shift this to Kruskal.java too. But if there are any concerns do bring it up!